engine/api/src/main/java/io/deephaven/engine/table/TableUpdate.java

```diff
@@ -52,25 +72,20 @@ && modified().isEmpty()
     }
 
     /**
-     * @return true if all internal state is initialized
+     *
```

Comment:
Analyze whether this TableUpdate is structurally valid, i.e. its accessors will return non-{@code null}, usable data structures. This does not test for usage outside of the appropriate updating phase.